FhirToDataLake/synapse/SynapseValidator.py

```diff
@@ -0,0 +1,194 @@
+import os
+import sys
+import argparse
+import concurrent.futures
+import json
+import requests
+import pyodbc
+import pandas as pd
+import time
+from urllib import parse as urlparse
+from datetime import datetime
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--synapse_workspace", required=True, help="Name of Synapse workspace.")
+parser.add_argument("--fhir_server_url", required=True, help="Fhir server url.")
+parser.add_argument("--schemas_path", required=True, help="Schemas directory path.")
+parser.add_argument("--database", default='fhirdb', help="Name of database.")
+parser.add_argument("--sql_username", help="SQL username.")
+parser.add_argument("--sql_password", help="SQL password.")
+args = parser.parse_args()
+
+sql_server_endpoint = args.synapse_workspace + "-ondemand.sql.azuresynapse.net"
+database = args.database
+schema_collection_directory = args.schemas_path
```

Comment:
Nit: Keep the consistency of variable names, e.g., `schema_directory = args.schema_directory`

Reply:
Updated parameter and variable both to ```schema_directory```